src/뉴스클러스터링.js

```diff
@@ -0,0 +1,68 @@
+export default function solution(str1, str2) {
+  const sliceList1 = [];
+  const sliceList2 = [];
+
+  for (let i = 0; i < str1.length - 1; i++) {
+    const slice1 = str1.slice(i, i + 2);
```

Comment:
slice로 배열을 자르기 보다 단순히 문자열을 더하여 처리하는 방법은 어떨까요?

`    const character = str[i] + str[i + 1];
`

Reply:
훨씬 더 직관적이네요! 감사합니다~

Reply:
```suggestion
    const slice1 = str1.substring(i, i + 2);
```
이렇게 내장메서드를 활용하는 방법도 있을 꺼 같습니다!

Reply:
감사합니다 준형님! 그런데 혹시 어떤 차이가 있을까요?? slice는 시간 복잡도가 늘어날까요??